mite_selenium/__init__.py

```diff
@@ -43,19 +47,41 @@ def _spec_import_if_none(self, config_option):
 
     def _start(self):
         self._context.browser = self
-        self._remote = Remote(
-            desired_capabilities=self._capabilities,
-            command_executor=self._command_executor,
-            browser_profile=self._browser_profile,
-            proxy=self._proxy,
-            keep_alive=self._keep_alive,
-            file_detector=self._file_detector,
-            options=self._options,
-        )
+        try:
+            if self._is_selenium_wire:
+                addr = socket.gethostbyname(socket.gethostname())
+                self._remote = SeleniumWireRemote(
+                    desired_capabilities=self._capabilities,
+                    command_executor=self._command_executor,
+                    browser_profile=self._browser_profile,
+                    proxy=self._proxy,
+                    keep_alive=self._keep_alive,
+                    file_detector=self._file_detector,
+                    options=self._options,
+                    seleniumwire_options={
+                        **self._seleniumwire_options,
+                        **{"addr": addr},
+                    },
+                )
+                logger.debug(f"Selenium-wire machine IP: {addr}")
+            else:
+                self._remote = SeleniumRemote(
+                    desired_capabilities=self._capabilities,
+                    command_executor=self._command_executor,
+                    browser_profile=self._browser_profile,
+                    proxy=self._proxy,
+                    keep_alive=self._keep_alive,
+                    file_detector=self._file_detector,
+                    options=self._options,
+                )
+        except Exception as e:
```

Comment:
This catch-all exception handling is a bit strange to see -- what kinds of exceptions do we expect to see?  Why doesn't the vanilla exception handler print them out correctly?